machine_learning_hep/analysis/analyzerdhadrons_mult.py

```diff
@@ -507,12 +511,19 @@ def makenormyields(self):
             lfile = TFile.Open(self.n_filemass)
             hNorm = lfile.Get("hEvForNorm_mult%d" % imult)
             normfromhisto = hNorm.GetBinContent(1)
+
+            lvar2_binmincorr = self.lvar2_binmin[imult]
+            lvar2_binmaxcorr = self.lvar2_binmax[imult]
+            if len(self.p_nbin2) > 2:
+                if self.lvar2_binmax[1] == self.lvar2_binmin[2]:
```

Comment:
This doesn't look right.
Should it be something like this?
`if imult < self.p_nbin2 - 1:`
`    if self.lvar2_binmax[imult] == self.lvar2_binmin[imult + 1]:`

Reply:
Hi Vit, thanks! I opened this PR quickly without testing it yet. Should be indeed like you wrote. @ginnocen, please note you need this fix when you continue to debug.